base/docs/basedocs.jl

```diff
@@ -821,6 +821,76 @@ The singleton type containing only the value `Union{}`.
 """
 Core.TypeofBottom
 
+"""
+    Core.Type{T} where T
+
+`Core.Type` is an abstract type which has all type objects as its instances.
+The only instance of the singleton type `Core.Type{T} where T` is the object
```

Comment:
This should say `Type{T}`, so the `T` is free.

Reply:
```suggestion
The only instance of the singleton type `Core.Type{T}` is the object
```